Cue/app/tabs/library/DeckView.js

```diff
@@ -254,6 +254,7 @@ class DeckView extends React.Component {
           onScroll={this._onScroll}>
           <DeckViewInfoHeader
             deck={this.state.deck}
+            key={this.state.deck.share_code}
```

Comment:
TBH I have no idea why `DeckViewInfoHeader` doesn't re-render when the deck is updated. It looks like it's receiving a stale copy of the deck, but I haven't been able to figure out why.